home/.chezmoi.yaml.tmpl

```diff
@@ -140,6 +140,7 @@ data:
     docker:
       doRegion: nyc1
     domain: "{{ $domain }}"
+    elastic_fleet_server: "https://es-fleet.megabyte.space:8220"
```

Comment:
Let's make this equal to `elastic-fleet.{{ $PUBLIC_SERVICES_DOMAIN }}:8220` instead.

Reply:
And name the data key `elasticFleetServer` instead to be consistent with capitalization

Reply:
Hey @enggnr --- I'm still seeing the `elasticFleetServer`

Reply:
Hey @enggnr -- can we use the same method we're using for etcd / glusterfs with SRV records?